rai/node/node.cpp

```diff
@@ -791,6 +811,36 @@ void rai::vote_processor::process_loop ()
 			lock.lock ();
 			active = false;
 			condition.notify_all ();
+
+			if (log_this_iteration)
+			{
+				end_time = std::chrono::steady_clock::now ();
+				elapsed_time = end_time - start_time;
+				elapsed_time_ms = std::chrono::duration_cast<std::chrono::milliseconds> (elapsed_time);
+				elapsed_time_ms_int = elapsed_time_ms.count ();
+	
+				if (elapsed_time_ms_int < 100)
+				{
+					/*
+					 * If the time spent was less than 100ms then
+					 * the results are probably not useful as well,
+					 * so don't spam the logs.
+					 */
+					log_this_iteration = false;
+				}
+			}
+			if (log_this_iteration)
+			{
+				if (elapsed_time_ms_int == 0)
```

Comment:
Seems like this won't happen since we're only logging if > 100 ms.
As a result, BOOST_LOG can be moved to a flipped `if (elapsed_time_ms_int < 100)` to simplify the code.

Reply:
I left it here in case we make the limits configurable